bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,226 +35,229 @@ Location,Garden,Outdoor,Garden,Gardens,
-Equipment,HeatDetector,FireDetector,Heat Detector,,
-Equipment,SmokeDetector,FireDetector,Smoke Detector,Smoke Detectors,
-Equipment,GasMeter,Sensor,Gas Meter,,
-Equipment,GlassBreakDetector,Sensor,Glass Break Detector,,
-Equipment,HumiditySensor,Sensor,Humidity Sensor,Hygrometer,
-Equipment,IlluminanceSensor,Sensor,Illuminance Sensor,,
-Equipment,LeakSensor,Sensor,Leak Sensor,,
-Equipment,OccupancySensor,Sensor,Occupancy Sensor,,
-Equipment,MotionDetector,OccupancySensor,Motion Detector,"Motion Detectors, Motion Sensor, Motion Sensors",
-Equipment,TemperatureSensor,Sensor,Temperature Sensor,,
-Equipment,VibrationSensor,Sensor,Vibration Sensor,,
-Equipment,WaterMeter,Sensor,Water Meter,,
-Equipment,WaterQualitySensor,Sensor,Water Quality Sensor,,
-Equipment,WeatherStation,Sensor,Weather Station,,
-Equipment,Siren,,Siren,Sirens,
-Equipment,Smartphone,,Smartphone,"Smartphones, Phone, Phones",
-Equipment,Tool,,Tool,,
-Equipment,Tracker,,Tracker,,
-Equipment,Valve,,Valve,Valves,
-Equipment,Vehicle,,Vehicle,,
-Equipment,Car,Vehicle,Car,,
-Equipment,VoiceAssistant,,Voice Assistant,Voice Assistants,
-Equipment,WebService,,Web Service,Web Services,
-Equipment,WeatherService,WebService,Weather Service,Weather Services,
-Equipment,Wellness,,Wellness,,
-Equipment,Chlorinator,Wellness,Chlorinator,,
-Equipment,Jacuzzi,Wellness,Jacuzzi,"Spa, Hot Tub, Whirlpool",
-Equipment,PoolCover,Wellness,Pool Cover,,
-Equipment,PoolHeater,Wellness,Pool Heater,,
-Equipment,Sauna,Wellness,Sauna,Steam Room,
-Equipment,Shower,Wellness,Shower,,
-Equipment,SwimmingPool,Wellness,Swimming Pool,"Swimming Pool, Pool",
-Equipment,WhiteGood,,White Good,White Goods,
-Equipment,AirFryer,WhiteGood,Air Fryer,,
-Equipment,CoffeeMaker,WhiteGood,Coffee Maker,"Coffee Makers, Coffee Machine, Coffee Machines",
-Equipment,Cooktop,WhiteGood,Cooktop,Hob,
-Equipment,Dishwasher,WhiteGood,Dishwasher,Dishwashers,
-Equipment,Dryer,WhiteGood,Dryer,"Dryers, Tumble Dryer, Tumble Dryers",
-Equipment,FoodProcessor,WhiteGood,Food Processor,,
-Equipment,Freezer,WhiteGood,Freezer,Freezers,
-Equipment,Fryer,WhiteGood,Fryer,,
-Equipment,IceMaker,WhiteGood,Ice Maker,,
-Equipment,Microwave,WhiteGood,Microwave,,
-Equipment,Mixer,WhiteGood,Mixer,,
-Equipment,Oven,WhiteGood,Oven,Ovens,
-Equipment,Range,WhiteGood,Range,,
-Equipment,Refrigerator,WhiteGood,Refrigerator,Refrigerators,
-Equipment,Toaster,WhiteGood,Toaster,Toaster Oven,
-Equipment,WashingMachine,WhiteGood,Washing Machine,Washing Machines,
-Equipment,Window,,Window,Windows,
-Equipment,WindowCovering,,Window Covering,,
-Equipment,Blinds,WindowCovering,Blinds,"Rollershutter, Rollershutters, Roller shutter, Roller shutters, Shutter, Shutters",
-Equipment,Drapes,WindowCovering,Drapes,Curtains,
-Equipment,Zone,,Zone,,
-Equipment,AlarmZone,Zone,Alarm Zone,,
+Property,Brightness,,Brightness,,Brightness of a light source
+Property,Channel,,Channel,,Radio or TV channel selection
+Property,Color,,Color,,Color of a light source
```

Comment:
```suggestion
Property,Color,,Color,,
```
This may apply to other things, e.g. Printer color, so perhaps just drop the desc